### ISSUE TITLE
Check control master handling in PR #435

### ISSUE THREAD

**Issue by hanke** [datalad/datalad#455]
Log:

````
% datalad create-publication-target-sshwebserver ssh://localhost/tmp/remoteloc/
mih@localhost's password: 
% datalad create-publication-target-sshwebserver ssh://localhost/tmp/remoteloc/ locremote
mih@localhost's password: 
2016-05-23 12:52:38,579 [ERROR  ] Target directory /tmp/remoteloc already exists. [create_publication_target_sshwebserver.py:__call__:241] (RuntimeError) (main.py:235)
% datalad create-publication-target-sshwebserver ssh://localhost/tmp/remoteloc/ locremote -f
mih@localhost's password: 
ControlSocket /var/run/user/1000/datalad/localhost already exists, disabling multiplexing
````

Looks like the 2nd command did not properly clean up upon exit.

**Comment by hanke** [comment]
Follow-up using the state of the previous post:

````
% datalad create-publication-target-sshwebserver ssh://localhost/tmp/remoteloc/ locremote -f
mih@localhost's password: 
% git branch -a
  git-annex
* master
  remotes/origin/HEAD -> origin/master
  remotes/origin/git-annex
  remotes/origin/master
````

It did not create the remote/sibling config that was promised in the docs.

**Comment by hanke** [comment]
I take the last one back. The config is there, but given nothing was published/fetched, it isn't shown above.

**Comment by bpoldrack** [comment]
If it's only about the first one now, it's fixed in PR #435
